app/javascript/react/components/GameTile.js

```diff
@@ -9,7 +10,7 @@ const GameTile = (props) => {
         <h6 className = "categories">{props.categories}</h6>
         <hr/>
         <p>{props.description}</p>
-        <a href='#' className="button view-game small-12 small-centered columns"> View Game </a>
+        <Link to={"games/"+props.id} className="button view-game small-12 small-centered columns">View Game</Link>
```

Comment:
You'll want to make sure you're using an absolute path in every one of your links (and fetches!), which is to say that here, you should have `"/games/" + props.id` with the slash in front of games. This makes sure it's not RELATIVE, or tacking onto whatever path they're already on.